.pre-commit-config.yaml

```diff
@@ -145,6 +145,26 @@ repos:
               echo;
               exit 1;
             } || true
+      - id: check-logger-usage
+        name: Check for proper logger usage (use llama_stack.log instead)
+        entry: bash
+        language: system
+        types: [python]
+        pass_filenames: true
+        args:
+          - -c
+          - |
+            grep -EnH '^[^#]*\b(import logging|from logging\b)' "$@" | grep -v '# allow-direct-logging' && {
```

Comment:
this should support github output, just like https://github.com/meta-llama/llama-stack/blob/main/scripts/check-workflows-use-hashes.sh